x/bounty/client/cli/tx.go

```diff
@@ -183,3 +185,59 @@ func NewSubmitFindingCmd() *cobra.Command {
 
 	return cmd
 }
+
+func NewWithdrawalFindingCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "withdrawal-finding",
+		Short: "withdrawal the specific finding",
```

Comment:
Use `withdrawal-finding 1 ` instead of  `withdrawal-finding --finding-it 1 `, it may be more user-friendly